client/me/profile/style.scss

```diff
@@ -24,6 +24,10 @@
 	}
 }
 
+.profile__settings .profile__public-url {
```

Comment:
It'd be nice if we could avoid local CSS like this.

Reply:
+1. Some utility classes would be helpful for cases like this. AFAIK we don’t currently have any though.